db/migrate/20200312095335_add_fields_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :is_phone_enabled, :boolean, default: true
+    add_column :users, :facebook, :string, default: 'facebook.com/zuck'
```

Comment:
dodałem taki default jak ty, ale raczej nie chcesz żeby pół krakostopu miało taki adres facebooka :D

Reply:
będzie to trzeba zmienić potem, jak już odkomentujesz tą linijkę